package-lock.json

```diff
@@ -6666,6 +6666,15 @@
         "node": ">=0.10.0"
       }
     },
+    "node_modules/@vue/cli-plugin-unit-jest/node_modules/is-wsl": {
```

Comment:
Revert the changes in package-lock file, as we don't push lock file until we have made some changes in the package.json file.